docs/subvolume.md

```diff
@@ -9,47 +9,47 @@ and delete them without impacting other resources and attached volumes.
 The subvolume command will require the following sub commands:
 * `ls` : [ls](#ls) lists all the subvolumes
     * `--stale`: lists only stale subvolumes
-* `delete <subvolumes> <filesystem> <subvolumegroup>`:
-    [delete](#delete) stale subvolumes as per user's input.
-    It will list and delete only the stale subvolumes to prevent any loss of data.
-    * subvolumes: comma-separated list of subvolumes of same filesystem and subvolumegroup.
-    * filesystem: filesystem name to which the subvolumes belong. 
-    * subvolumegroup: subvolumegroup name to which the subvolumes belong.
+* `delete <filesystem> <subvolume> <subvolumegroup>`:
+    [delete](#delete) stale subvolume as per user's input.
+    It will delete only the stale subvolume to prevent any loss of data.
+    * subvolume: subvolume name.
+    * filesystem: filesystem name to which the subvolume belong.
+    * subvolumegroup(optional): subvolumegroup name to which the subvolume belong.
+      * By default the subvolumegroup is "csi". 
```

Comment:
```suggestion
```

Reply:
Since the field is option, IMO its good to mention the default value??